### ISSUE TITLE
[Overview - !!] Title

### ISSUE THREAD

**Issue by shawna-slh** [w3c/wai-older-users#3]
Keeping in mind **criteria for the title**:
* Clearly communicate the purpose of the resource: To tell people looking for guidelines for meeting older user needs that they can/should use WCAG (and UAAG) to cover older users, too!
* Terms people will look for in skimming the navigation and for SEO -- probably "older"; maybe "senior", "ageing"
* Would be good to have nice short title for nav, and can have longer title or subtitle for clarification
* Nav will be in context on WAI website, but doc title can be in other contexts, e.g., search engine results, so it should be clear stand-alone.
* @@ other criteria?

fyi, previous title was:
> **Web  Accessibility and Older People:** _[short title for nav]_
Meeting the Needs of Ageing Web Users  _[subtitle for additional clarification and SEO]_

If we did want to make it more explanatory, I think it would be something like:

1. **Meeting the Needs of Older Users through Web Accessibility Guidelines**

Although that seems long and I'm not sure what the short title would be for the navigation?

Or this is a bit more concise:
2. **Meeting Older Users' Needs through Web Accessibility Guidelines**
In the nav, since it's in context of WAI website, shortened title could be "Meeting Older Users' Needs"

Other brainstorms:
3. **Web Accessibility Guidelines to Meet Older User Needs** _(con: not front-loaded with "older user")_
4. **Designing for Older Users Using Web Accessibility Guidelines** _(nav could be just "Designing for Older Users")_

**Terminology**
* I'm not sure of "Older" vs "Ageing" if we only use one word. (We're *all* ageing! ;-)
* "Guidelines" or "Standards"? Which would people look for most, and be most happy finding?
 "Guidelines/Standards" is most accurate, although maybe not worth the complexity?

Thoughts???

**Comment by shawna-slh** [comment]
After discussion with Vicki, looking at redesigned IA, Vicki and I suggest:

> **Older Users and Web Accessibility:**
Meeting the Needs of Ageing Web Users
